.github/workflows/docker-publish.yml

```diff
@@ -7,48 +7,36 @@ on:
     tags:
       - '**'
   workflow_dispatch:
-    inputs:
-      platform:
-        description: 'Platform'
-        default: 'linux/amd64'
-        required: false
 
 jobs:
   docker:
     name: "Docker Image"
     runs-on: ubuntu-latest
     steps:
-
     - name: Checkout Project
       uses: actions/checkout@v3
-
     - name: Docker meta
       id: docker_meta
-      uses: docker/metadata-action@v4
+      uses: docker/metadata-action@v3
       with:
         images: |
-          wipac/skydriver
+          ghcr.io/WIPACRepo/SkyDriver
         tags: |
-            # release tags
-            type=semver,pattern={{major}},enable=${{ github.ref_type == 'tag' }}
-            type=semver,pattern={{major}}.{{minor}},enable=${{ github.ref_type == 'tag' }}
-            type=semver,pattern={{major}}.{{minor}}.{{patch}},enable=${{ github.ref_type == 'tag' }}
-
-    - name: Login to Docker Hub
-      uses: docker/login-action@v2
+            type=ref,event=branch
+            type=semver,pattern={{major}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}.{{minor}}.{{patch}}
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
       if: ${{ github.event_name != 'pull_request' }}
```

Comment:
I think it can't be a pull request, since the push selector removes them.  So this line seems useless.